packages/languages/src/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * External dependencies
+ */
+import values from 'lodash/values';
+
+/**
+ * Internal dependencies
+ */
+import data from './languages-meta.json';
+
+export const languages = values( data );
```

Comment:
Not really required, but might be better to have the languages as default export.